ui/src/pages/stop-pending/stop-pending.css

```diff
@@ -0,0 +1,39 @@
+Copy code
+/* src/pages/stop-pending/StopPending.css */
+#stop-pending.container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    text-align: center;
+    font-family: Arial, sans-serif;
+    padding-top: 50px; 
+}
+
+.spinner {
+  margin: 20px 0;
+  width: 40px;
+  height: 40px;
+  border: 4px solid rgba(0, 0, 0, 0.1);
+  border-top: 4px solid #a020f0; /* Purple color for the spinner */
+  border-radius: 50%;
+  animation: spin 1s linear infinite;
+}
+
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(360deg); }
+}
```

Comment:
Why is this needed? Doesn't the circular progress work on its own?

Reply:
That was a spinner before Mui spinner, removed